lineage/figures/figure8.py

```diff
@@ -36,7 +36,8 @@ def makeFigure():
     E = [Eone, Etwo, Ethree, Efour, Eone, Etwo, Ethree, Efour]
 
     # making lineages and finding AICs (assign number of lineages here)
-    AIC = [run_AIC(.1, e, 10, idx > 3) for idx, e in enumerate(E)]
+    Aic = [[run_AIC(.1, e, 10, idx > 3) for idx, e in enumerate(E)] for _ in range(10)]
+    AIC= list(map(list, zip(*Aic)))
```

Comment:
What is this line meant to do?

Reply:
If you mean line 40 that I added, honestly I tried to make it look like the form that Luka had written, because of the plotting. It kind of reshapes this list of list of lists so the dimensions become more like what it used to be.

Reply:
If this is not a problem, I think this branch is ready @aarmey